tests/test_repo_synchronizer.py

```diff
@@ -103,6 +103,9 @@ def test_sync_process_(
     assert tag in tag_log
     assert hg_rev(hg_destination, branch) in tag_log
 
+    tag_author = hg_log(hg_destination, tag_branch, ["-T", "{author}"])
+    assert request_user in tag_author
```

Comment:
Can we assert the author value as an exact match instead?

Also side nit: I noticed this test is named `test_sync_process_`, seems like that should be `test_sync_process_tags`.

Reply:
It does both commits and tags. But the trailing `_` is unsightly.